src/ScopeCommand.php

```diff
@@ -40,6 +40,8 @@ protected function configure(): void
         $this->addOption('yes', 'y', InputOption::VALUE_NONE, 'Bypass confirmation.');
         $this->addOption('rename-references', null, InputOption::VALUE_NONE,
             'Rename references in the main repo after prefixing dependencies. (Note: this will destroy formatting of PHP code so is not enabled by default.)');
+        $this->addOption('ignore-platform-check', null, InputOption::VALUE_NONE,
+            'Remove the platform check when the new autoloader is generated.)');
```

Comment:
```suggestion
            'Remove the platform check when the new autoloader is generated.');
```

Reply:
Good catch! Thank you for spotting that :+1: